theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
I may rewrite GpuDimShuffle today.



NB: small remark: One of the problems when we rewrite ops is that we now need to allocate small arrays (`int* shape = (int*)malloc(params->size * sizeof(int))`), where we previously just hardcoded it (`int shape[%(size)s]`). It's quite regrettable, but I don't find a way to do it otherwise.